service/src/main/java/greencity/service/ubs/UBSClientServiceImpl.java

```diff
@@ -902,6 +902,7 @@ public UserProfileDto getProfileData(String uuid) {
                 .collect(Collectors.toList());
         userProfileDto.setAddressDto(addressDto);
         userProfileDto.setBotList(botList);
+        userProfileDto.setHasPassword(userRemoteClient.hasPassword().getHasPassword());
```

Comment:
Does it mean that hasPassword should always be false? Or when we change it to true?

Reply:
We change it to true when we know that user has password set